compiler/resolution/functionResolution.cpp

```diff
@@ -1622,23 +1622,20 @@ moreSpecific(FnSymbol* fn, Type* actualType, Type* formalType) {
 
 static bool
 computeActualFormalAlignment(FnSymbol* fn,
```

Comment:
Another case in which the procedure formals seem to me to be in a "dis-organized" sequence.